### PR TITLE
Fixes Issue #9. Specify output path

### PR DESCRIPTION
**Pull request by yinzara**

### CODE REVIEW

**Comment by srikrsna** [comment]
Thank you for the PR. Although I still don't understand the nature of the problem since I am unable to replicate it on my machine. I'll merge this as this is completely optional.

File: README.md [hunk 1]
```diff
@@ -53,6 +53,16 @@ It is very useful to ignore XXX* fields in protobuf generated messages. The go p
     	--gotag_out=xxx="graphql+\"-\" bson+\"-\"":. example/example.proto
 ```
 
+## Output to a directory other than the current directory
```
Comment (srikrsna) [inline README.md:56]: Can you change this to an issue experienced with prototool as not a lot (including me despite using prototool) are not facing the issue. If you can help me replicate I'll be happy to add in the situation

Comment (yinzara) [reply README.md:56]: Look at the comments I added to Issue #9   It gives extremely simple reproduction steps and the cause.

Comment (yinzara) [reply README.md:56]: Also once you merge, can you tag so I can use a go mod replace?

Comment (bviolier) [reply README.md:56]: Are there any plans to still merge this? I would like to use this as well :-)

**Pull request merged**